returnn/dataset.py

```diff
@@ -1,14 +1,15 @@
-__all__ = ["ExtractDatasetMeanStddevJob"]
+__all__ = ["ExtractDatasetMeanStddevJob", "get_returnn_length_hdfs"]
```

Comment:
IIRC, we usually have helper functions in separate files than the job definitions. Would we want to move this to a separate file, e.g. `returnn/helpers.py`?

What is - in general - our take on jobs/helpers/pipeline code in i6_core/i6_experiments?

Reply:
Helper functions which are required by the job itself, are located in the same file: https://github.com/rwth-i6/i6_core/blob/main/features/filterbank.py

Helper functions which ease the usage of a job are in a separate file.

So I think these functions #294 and #297 both fall in the second category..

`i6_core`: jobs
`i6_experiments`: pipeline

helpers: depends?! -> helpers in `i6_core` seem to mainly deal with contruction of flow nodes and rasr configs (more or less). `i6_experiments` more varied. at least this is my feeling..  I am not sure where to exactly draw the line.. 

helpers for a job -> `i6_core`
helpers for a pipeline -> `i6_experiments`
????

Reply:
I have a very similar feeling and therefore my gut feeling would have placed this (#270) somewhere in i6_experiments/common. This is a week preference

#294 I am also ok with having this as an initial function in lib/hdf.py.. I think we already thought about creating such a lib..

I'll leave the final decision to you guys

Reply:
So I think lib/hdf.py will get a bit more content from @JackTemaki soon iirc. 
The other helper could ofc. be moved into a different file but I would argue its directly returnn dataset related and right now the file is not really long. Helpers can afterwards once files become too large easily be moved. Creating a helper file for a single function to me feels like making things too complicated (esp. as a general guideline). 
That being said I dont really mind either way, this might just be a case where it in the end doesnt really matter.

Reply:
I would say this is (In contrast to #294) very specific to a certain kind of pipeline. Lets change this PR here to only update the `ReturnnForwardJob` code, and put the rest in `i6_experiments/common`. Unfortunately it is still not clear how and where "Returnn" setup related code will be put and structured, but I will try to figure that out as soon as possible.